vars/main.yml

```diff
@@ -23,6 +23,22 @@ core:
   upf:
     ip_prefix: "192.168.252.0/24"
     iface: "access"
+    # when using 'dpdk' mode please make sure atleast two VF devices are
+    # created out of 'data_iface'
+    mode: af_packet          # Options: af_packet or dpdk
+    # set to true if mode is dpdk. This will enable the required resources
+    # like hugepage, sriov, etc...
+    enable_resources: false
+    # configure the PCI address of VF's which created out of 'data_iface'.
+    # It will be used for N3, N6 interfaces. Configure these values only if
+    # mode is set to 'dpdk' otherwise just leave this parameter as it is.
+    pci_devices: [0000:b1:01.0, 0000:b1:01.1]
+    cniPlugin: macvlan     # macvlan (for af_packet) or vfioveth (for dpdk mode)
+    # set to true if mode is dpdk. This is INTEL NIC's specific feature
+    # (supported in E810 series) which helps to improve UPF packet processing.
+    # For more details on DDP feature and its usage please refer
+    # "https://www.intel.com/content/www/us/en/download/19660/intel-ethernet-800-series-dynamic-device-personalization-ddp-for-telecommunication-comms-package.html"
+    ddp: false
```

Comment:
Actually, I am wondering why you have to expose all these variables in the `main.yml` (`main-sriov.yml`) file? Why not just having them in the `deps/5gc/core/templates/sdcore-5g-values.yaml` file

Reply:
I agree. These are similar to other parameters defined in `sdcore-5g-values.yaml`.  It's not a bright line separating what gets defined where, but that seems like the right place.

Reply:
done